gravitee-apim-console-webui/src/management/api/portal/user-group-access/members/api-portal-members.harness.ts

```diff
@@ -61,6 +62,26 @@ export class ApiPortalMembersHarness extends ComponentHarness {
     return this.getMemberRoleSelectForRowIndex(rowIndex).then((select) => select.isDisabled());
   }
 
+  async getMemberDeleteButton(rowIndex: number): Promise<MatButtonHarness> {
+    const deleteCell = await this.getMemberDeleteCell(rowIndex);
+    return deleteCell.getHarness(MatButtonHarness);
+  }
+
+  async getMemberDeleteCell(rowIndex: number): Promise<MatCellHarness> {
+    const rows = await this.getTableRows();
+    expect(rows.length).toBeGreaterThan(rowIndex);
+    const deleteCell = await rows[rowIndex].getCells({ columnName: 'delete' });
+    expect(deleteCell.length).toEqual(1);
+    return deleteCell[0];
+  }
+
+  async isMemberDeleteButtonVisible(rowIndex: number): Promise<boolean> {
+    return this.getMemberDeleteCell(rowIndex).then(async (cell) => {
+      const harnesses = await cell.getAllHarnesses(MatButtonHarness);
```

Comment:
Why get all harnesses and not a specific button harness?

Reply:
because getHarness throw if not found. 
So two way to check no button display find a list of 0 element or add a try catch the promise
Personally i prefer the `getAllHarnesses`